djangoevents/tests/test_domain.py

```diff
@@ -1,9 +1,9 @@
-from ..domain import BaseEntity, DomainEvent
+from ..domain import BaseEntity, DomainEvent, list_aggregate_events, list_concrete_aggregates
 
 import pytest
 
 
-class SampleEvent(BaseEntity):
+class SampleEntity(BaseEntity):
```

Comment:
Let's leave his one for now untill we drop support for BaseEntity. It inherits from BaseAggregate which give us code coverage for both classess at the same time.